core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -104,7 +111,7 @@ private[data] sealed trait OneAndInstances extends OneAndLowPriority2 {
     }
 
   implicit def oneAndSemigroup[F[_]: MonadCombine, A]: Semigroup[OneAnd[F, A]] =
-    oneAndSemigroupK.algebra
+    oneAndSemigroupK(MonadCombine[F]).algebra
```

Comment:
sorry I misread your comment @ceedubs, you were right `oneAndSemigroupK[F]` is enough to disambiguate.